### ISSUE TITLE
Undo Bug With Multiple Empty Lines

### ISSUE THREAD

**Issue by ryabouza** [justy989/ce_old#64]
Undo doesn't undo everything when entering multiple blank lines.

To reproduce:
o
enter
enter
enter
enter
enter
escape
u
u
u
u
u

**Comment by robber-m** [comment]
NOTE: likely an issue with the following case in `ce_commit_insert_mode_changes`:

``` c
if(config_state->start_insert.x == config_state->original_start_insert.x &&
             config_state->start_insert.y == config_state->original_start_insert.y)
```

Could be an issue with the `ce_dupe_string` call in this case.

**Comment by justy989** [comment]
should be fixed in #76
